uxarray/plot/dataarray_plot.py

```diff
@@ -61,3 +71,159 @@ def datashade(uxda: UxDataArray,
         _cmap = cmap
 
     return tf.shade(aggregated, cmap=_cmap, **kwargs)
+
+
+def _point_raster(uxda: UxDataArray,
+                  backend: Optional[str] = "bokeh",
+                  pixel_ratio: Optional[float] = 1.0,
+                  dynamic: Optional[bool] = False,
+                  precompute: Optional[bool] = True,
+                  projection: Optional[ccrs] = None,
+                  width: Optional[int] = 1000,
+                  height: Optional[int] = 500,
+                  colorbar: Optional[bool] = True,
+                  cmap: Optional[str] = "Blues",
+                  aggregator: Optional[str] = "mean",
+                  interpolation: Optional[str] = "linear",
+                  npartitions: Optional[int] = 1,
+                  **kwargs):
+
+    if uxda.face_centered():
+        # data mapped to face centroid coordinates
+        data_mapping = "center"
+        lon = uxda.uxgrid.Mesh2_face_x.values
+        lat = uxda.uxgrid.Mesh2_face_y.values
+    elif uxda.node_centered():
+        # data mapped to face corner coordinates
+        data_mapping = "corner"
+        lon = uxda.uxgrid.Mesh2_node_x.values
+        lat = uxda.uxgrid.Mesh2_node_y.values
+    else:
+        raise ValueError(
+            "Issue with data. It is neither face-centered nor node-centered!")
+
+    data_values = uxda.values
```

Comment:
`data_values` being called only twice in the rest; let us just use `uxda.values` to help increase readability of the code by avoiding defining too many variables.